src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+                    <td>
+                        0 = Have been polled before.
+                        <br></br>
+                        1 = First poll after power up.
+                    </td>
+                </tr>
+                <tr>
+                    <td>0 - Bit 1</td>
+                    <td>
+                        0 = Not Going to Sleep.
+                        <br></br>
+                        1 = Going to Sleep.
+                    </td>
+                </tr>
+                <tr>
+                    <td>0 - Bit 2</td>
+                    <td>
+                        0 = Normal reply.
+                        <br></br>
+                        1 = First poll response after wake up.
+                    </td>
+                </tr>
+                <tr>
+                    <td>0 - Bit 3-7</td>
+                    <td>Reserved</td>
+                </tr>
+                <tr>
+                    <td>&nbsp;</td>
+                    <td>The following Bytes are sent only on first poll-response</td>
+                </tr>
+                <tr>
+                    <td>1</td>
+                    <td>Firmware Version Major</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Firmware Version Minor</td>
+                </tr>
+                <tr>
+                    <td>3-4</td>
+                    <td>2 Most significant bytes of Serial Number</td>
+                </tr>
+                <tr>
+                    <td>5</td>
+                    <td>
+                        Reset Reason
+                        <br></br>
+                        0 = Power On Reset
+                        <br></br>
+                        1 = Reset Command
+                        <br></br>
+                        2 = New Firmware
+                        <br></br>
+                        All others reserved.
+                    </td>
+                </tr>
+                <tr>
+                    <td>6</td>
+                    <td>Reserved</td>
+                </tr>
+                <tr>
+                    <td>7</td>
+                    <td>
+                        150 to 199 
+                        <br></br>
+                        IOX ID. Please contact Geotab to get one assigned.
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <p>When the "Go to Sleep" command is received, but before actually going to sleep, the devices will indicate they are going to sleep through the indicated bit. This bit is cleared on wakeup.</p>
+        <h3>Additional Info (0x03)</h3>
+        <p>Sent by the IOX after an ACK for the first poll is received. This message is not strictly required for operation. However, sending of this message is required if any version information is to be reported, including: Product, Hardware, Firmware Major, Firmware Minor, or Version Control.</p>
+        <h4>Payload - Additional Info</h4>
+        <table>
```

Comment:
For the time being, Joseph mentioned we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on